src/app/hooks/useTips.ts

```diff
@@ -1,15 +1,20 @@
 import { useAccount } from "~/app/context/AccountContext";
 import { useSettings } from "~/app/context/SettingsContext";
-import { isAlbyAccount } from "~/app/utils";
+import { isAlbyAccount, isAlbyLNDHubAccount } from "~/app/utils";
 import { TIPS } from "~/common/constants";
 
 const DEFAULT_TIPS = [TIPS.TOP_UP_WALLET, TIPS.MNEMONIC];
 
-export const filterTips = (closedTips: TIPS[], connector = "") => {
+export const filterTips = (closedTips: TIPS[], connector = "", alias = "") => {
```

Comment:
I don't think filterTips should have to know how to deal with `connector` and `alias`. We should try limit this as much as possible.

rather than doing the calculation inside I wonder if it should pass these parameters as boolean values (isAlbyAccount, isAlbyLNDHubAccount). 

Note: I think when we add the onchain address for the Alby OAuth connector, then this tip might link to a different place for the new one (like receive screen in the extension)?

Reply:
did those changes kindly review